pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["flit"]
+build-backend = "flit.buildapi"
+
+[tool.flit.metadata]
+module = "gidgethub"
+author = "Brett Cannon"
+author-email = "brett@python.org"
+home-page = "https://github.com/brettcannon/gidgethub"
+requires = ["uritemplate>=3.0.0"]
+requires-python = ">=3.6"
+license = "Apache"
+keywords = "github sans-io async"
+description-file = "README.rst"
+classifiers = ["Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.6",
+]
+
+[tool.flit.metadata.requires-extra]
+test = ["pytest>=3.0.0", "pytest-asyncio"]
+doc = ["sphinx"]
+aiohttp = ["aiohttp"]
+treq = ["treq", "twisted[tls]"]
+tornado = ["tornado"]
+setup = ["pytest-runner>=2.11.0"]
```

Comment:
I think this is no longer necessary as this was just for setuptools in order to make `setup.py test` function.

Reply:
Removed.